pkg/cmd/grafana-cli/services/api_client.go

```diff
@@ -101,15 +101,15 @@ func (client *GrafanaComClient) DownloadFile(pluginName string, tmpFile *os.File
 	}()
 
 	w := bufio.NewWriter(tmpFile)
-	h := md5.New()
+	h := sha256.New()
 	if _, err = io.Copy(w, io.TeeReader(bodyReader, h)); err != nil {
-		return errutil.Wrap("Failed to compute MD5 checksum", err)
+		return errutil.Wrap("Failed to compute sha256 checksum", err)
```

Comment:
Go errors should generally start with a lowercase letter:
```suggestion
		return errutil.Wrap("failed to compute SHA256 checksum", err)
```